OfflineMapBook/OfflineMapBook_NetFramework/Properties/Settings.Designer.cs

```diff
@@ -49,7 +49,7 @@ internal sealed partial class Settings : global::System.Configuration.Applicatio
         
         [global::System.Configuration.UserScopedSettingAttribute()]
         [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
-        [global::System.Configuration.DefaultSettingValueAttribute("http://runtime.maps.arcgis.com/sharing/rest")]
+        [global::System.Configuration.DefaultSettingValueAttribute("https://runtime.maps.arcgis.com/sharing/rest")]
```

Comment:
Can you confirm the app operates as expected after this change?

Reply:
I confirmed that the app works as it should after this change